pkg/apiserver/openapi/openapi.go

```diff
@@ -72,10 +73,15 @@ func GetOperationID(servePath string, r *restful.Route) (string, error) {
 			prefix = prefix + ToValidOperationID(strings.TrimSuffix(parts[1], ".k8s.io"), prefix != "")
 			if len(parts) > 2 {
 				prefix = prefix + ToValidOperationID(parts[2], prefix != "")
+				tags = append(tags, fmt.Sprintf("%s_%s", ToValidOperationID(parts[2], false), ToValidOperationID(strings.TrimSuffix(parts[1], ".k8s.io"), false)))
```

Comment:
Please don't use `.` as the separator. Group might have `.` in the name. (Though I remember you removed .k8s.io suffix)